wandb/sdk/data_types/html.py

```diff
@@ -16,19 +16,44 @@
 
 
 class Html(BatchableMedia):
-    """Wandb class for arbitrary html.
-
-    Args:
-        data: (string or io object) HTML to display in wandb
-        inject: (boolean) Add a stylesheet to the HTML object.  If set
-            to False the HTML will pass through unchanged.
-    """
-
     _log_type = "html-file"
 
-    def __init__(self, data: Union[str, "TextIO"], inject: bool = True) -> None:
+    def __init__(
+        self,
+        data: Union[str, "TextIO"],
+        inject: bool = True,
+        data_is_not_path: bool = False,
+    ) -> None:
+        """A class for logging HTML to W&B.
```

Comment:
This is a description of the class, not the initializer.

For the class docstring, I would say "HTML contents to upload to W&B." This is what will show up on hover in IDEs / notebooks if the user hovers over the type.

For the init docstring, the summary can be "Creates a W&B HTML object." The summary doesn't need to be much more descriptive because it's the initializer, and the rest of the description is more important.